pkg/vault/contracts/Authorizer.sol

```diff
@@ -50,6 +51,7 @@ contract Authorizer is IAuthorizer {
         uint256 executableAt;
     }
 
+    IAuthentication public immutable vault;
```

Comment:
Can we change this to private and have a `getVault` getter? This'll match the style used in other contracts.